src/rank_llm/rerank/rank_gpt.py

```diff
@@ -127,7 +127,7 @@ def run_llm(
         prompt: Union[str, List[Dict[str, str]]],
         current_window_size: Optional[int] = None,
     ) -> Tuple[str, int]:
-        model_key = "engine" if self.use_azure_ai else "model"
+        model_key = "model"
```

Comment:
why this change?

Reply:
This was Ronak

Reply:
@ronakice PTAL

Reply:
I think the engine is now just model? I was getting errors when I try to run it in the newest version.